### ISSUE TITLE
Differences between PhantomJS & Chrome

### ISSUE THREAD

**Issue by aral** [dalekjs/dalek#102]
Results differ between PhantomJS and Chrome while testing Stripe iframe functionality with the local version of the page at http://indietech.org/summit …

(Not a showstopper for us at the moment as we’re just going to use Chrome for this, but thought you’d like to know. I can only imagine they should both return the same result.) :)

With PhantomJS:

RUNNING TEST - "Stripe"
▶ OPEN http://localhost:8000/summit
▶ TYPE #name-1
▶ TYPE #email-1
✔ ENABLED Purchase tickets button is enabled.
▶ CLICK #purchase-tickets-button
▶ TOFRAME true
▶ WAITFORELEMENT
✘ EXISTS
0 EXPECTED: undefined
0 FOUND: false
0 MESSAGE: App view exists
▶ WAITFORELEMENT
✘ EXISTS
0 EXPECTED: undefined
0 FOUND: false
0 MESSAGE: Test Mode exists.
✘ TEST - "Stripe" FAILED

 1/3 assertions passed. Elapsed Time: 1 min 26.7 sec

---

With Chrome (35.0.x)

UNNING TEST - "Stripe"
▶ OPEN http://localhost:8000/summit
▶ TYPE #name-1
▶ TYPE #email-1
✔ ENABLED Purchase tickets button is enabled.
▶ CLICK #purchase-tickets-button
▶ TOFRAME true
▶ WAITFORELEMENT
✔ EXISTS App view exists
▶ WAITFORELEMENT
✔ EXISTS Test Mode exists.
✔ 3 Assertions run
✔ TEST - "Stripe" SUCCEEDED

---

The test:

``` coffeescript
module.exports =
    'Stripe': (test) ->
        test
            .open('http://localhost:8000/summit')
            .type('#name-1', 'Aral Balkan')
            .type('#email-1', 'aral@indiephone.eu')
            .assert.enabled('#purchase-tickets-button', 'Purchase tickets button is enabled.')
            .click('#purchase-tickets-button')
            .toFrame('.stripe_checkout_app')
            .waitForElement('.appView', 1000)
            .assert.exists('.appView', 'App view exists')
            .waitForElement('.testMode', 1000)
            .assert.exists('.testMode', 'Test Mode exists.')
            .done()
```

**Comment by danielgtsong** [comment]
@asciidisco yes, i am also getting an error for phantomjs and not for chrome.

Running tests
Running Browser: Google Chrome
OS: Mac OS X 10.10.2 x86_64
Browser Version: 44.0.2403.107

RUNNING TEST - "Verify movie offers on first title"
▶ OPEN http://int-lg.mgo.com/20/#startup
▶ WAIT 3000 ms
▶ SCREENSHOT ./build/screenshot/lg_guest/01.png
▶ EXECUTE 
✔ EXISTS The browse button exists
▶ CLICK ul.startup-footer-nav li:nth-of-type(1) a
▶ WAIT 3000 ms
▶ EXECUTE 
✔ EXISTS The first card exists
▶ WAIT 2000 ms
✔ 2 Assertions run
✔ TEST - "Verify movie offers on first title" SUCCEEDED

 2/2 assertions passed. Elapsed Time: 12.6 sec 

Running tests
Running Browser: PhantomJS
OS: mac 10.10 (Yosemite) 32bit
Browser Version: 1.9.8

RUNNING TEST - "Verify movie offers on first title"
▶ OPEN http://int-lg.mgo.com/20/#startup
▶ WAIT 3000 ms
▶ SCREENSHOT ./build/screenshot/lg_guest/01.png
▶ EXECUTE 
✔ EXISTS The browse button exists
▶ CLICK ul.startup-footer-nav li:nth-of-type(1) a
▶ WAIT 3000 ms
▶ EXECUTE 
✘ EXISTS
0 EXPECTED: undefined
0 FOUND: false
0 MESSAGE: The first card exists
▶ WAIT 2000 ms
✘ TEST - "Verify movie offers on first title" FAILED

 1/2 assertions passed. Elapsed Time: 16.89 sec 

very confused about this.
